examples/atlas-api-key/create-and-assign-pak-seperately/README.md

```diff
@@ -0,0 +1,12 @@
+# MongoDB Atlas Provider -- Create and Assign PAK sepeerately
```

Comment:
All three examples are really good but this whole area can be hard to understand if the UI is your main PAK experience.  In the UI one has organization keys and project keys - if one looks closely you can tell  all are really  org keys and are simply assigned to projects but that's not as clear as it is in the Admin API.  I think an intro README at the atlas-api-key folder level explaining this and why we provided 3 examples would be very useful.   It's not a blocker to this PR but I think would be a good follow up, perhaps for the docs team?  cc @Zuhairahmed